crown-court-proceeding/src/main/java/uk/gov/justice/laa/crime/crowncourt/service/RepOrderService.java

```diff
@@ -122,4 +129,95 @@ public ReviewResult getReviewResult(ApiIOJAppeal apiIOJAppeal) {
         return null;
     }
 
+    public ApiCrownCourtSummary determineCrownRepType(CrownCourtsActionsRequestDTO requestDTO) {
```

Comment:
Should we be calling this from the Proceedings Service in the `checkCrownCourtActions` function or will that be done as part of another ticket?

Reply:
LCAM-216 will handle the call to this method.